client/src/__tests__/Pages/CoursesPage/CoursesPage.test.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import renderer from "react-test-renderer";
 import { AppConfiguration } from "../../../components";
-import { CoursesPage } from "../../../pages";
+import { Courses } from "../../../pages";
 
 test("<CoursesPage />", () => {
```

Comment:
Can we rename this to `Courses` now?

Reply:
I'll make a ticket for refactoring common components as this is for structure :D